modules/cluster-logging-rn-5.4.11.adoc

```diff
@@ -0,0 +1,21 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-11_{context}"]
+= Logging 5.4.11
+This release includes link:https://access.redhat.com/errata/RHSA-2023:0632[OpenShift Logging Bug Fix Release 5.4.11].
```

Comment:
Getting 404 PAGE NOT FOUND for the errata link. Hope it is correct. Please check.